src/ConnectedMode/Resources.resx

```diff
@@ -141,6 +141,9 @@
   <data name="BranchProvider_CouldNotDetectGitRepo" xml:space="preserve">
     <value>[ConnectedMode/BranchMapping] Could not detect a git repo for the current solution</value>
   </data>
+  <data name="BranchProvider_FailedToCalculateMatchingBranch" xml:space="preserve">
+    <value>[ConnectedMode/BranchMapping] Failed to calculate closest local git branch, defaulting to server main branch.</value>
```

Comment:
It might be worth saying `... defaulting to Sonar main branch.` (or `Sonar server main branch`), just to make it absolutely clear.

Since the message is talking about local git branches then server branches, my first thought is that "server" means git server. I have to remind myself it doesn't.